python_ta/checkers/global_variables_checker.py

```diff
@@ -33,16 +33,19 @@ def __init__(self, linter=None) -> None:
         super().__init__(linter)
         self.import_names = []
 
+    @only_required_for_messages("forbidden-global-variables")
     def visit_global(self, node: nodes.Global) -> None:
         args = "the keyword 'global' is used on line {}".format(node.lineno)
         self.add_message("forbidden-global-variables", node=node, args=args)
 
+    @only_required_for_messages("forbidden-global-variables")
     def visit_assignname(self, node: nodes.AssignName) -> None:
         """Allow global constant variables (uppercase) and type aliases (type alias pattern), but issue messages for
         all other globals.
         """
         self._inspect_vars(node)
 
+    @only_required_for_messages("forbidden-global-variables")
```

Comment:
This is good, but you can add the decorator to the other `visit_` methods below as well (since they aren't needed if we aren't checking for `forbidden-global-variables`)